driver/csiplugin/controllerserver.go

```diff
@@ -618,6 +645,12 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 
 	glog.V(5).Infof("volume:[%v] - check if volume filesystem [%v] is mounted on GUI node of Primary cluster", scaleVol.VolName, scaleVol.VolBackendFs)
 	volFsInfo, err := scaleVol.PrimaryConnector.GetFilesystemDetails(scaleVol.VolBackendFs)
+
+	if isNewVolumeType || (scaleVol.IsFilesetBased && scaleVol.Tier != "") {
+		if err := cs.checkCGAndVolTierSupport(volFsInfo.Version); err != nil {
```

Comment:
We should be checking the scale version instead filesystem version here.

Reply:
Done

Reply:
@amdabhad we have to check fsversion for tier and scaleversion for CG

Reply:
Done